runtime/criusupport/criusupport.cpp

```diff
@@ -707,6 +708,20 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 			break;
 		}
 
+		if (NULL != vm->checkpointState.restoreArgsList) {
+			if (FIND_ARG_IN_ARGS_FORWARD(vm->checkpointState.restoreArgsList, OPTIONAL_LIST_MATCH, VMOPT_XTRACE, NULL) >= 0) {
+				RasGlobalStorage *j9ras = (RasGlobalStorage *)vm->j9rasGlobalStorage;
+				if ((NULL != j9ras) && (NULL != j9ras->criuRestoreInitializeTrace)) {
```

Comment:
if we fail to init trace for any reason there should be a message

Reply:
Set `J9NLS_JCL_CRIU_RESTORE_INITIALIZE_TRACE_FAILED` if `j9ras` or `j9ras->criuRestoreInitializeTrace` is `null`.